server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/Db2SqlLexer.g4

```diff
@@ -297,7 +297,7 @@ END_EXEC : E N D MINUSCHAR E X E C;
 ENDING : E N D I N G;
 ENFORCED : E N F O R C E D;
 ENVIRONMENT : E N V I R O N M E N T;
-EQUALCHAR : E Q U A L C H A R;
+EQUALCHARLITERAL : E Q U A L C H A R;
```

Comment:
E Q U A L C H A R L I T E R A L

Reply:
I am not sure If we have used it But it was conflicting with the same lexer rule in CICS 🤕

Reply:
> I am not sure If we have used it But it was conflicting with the same lexer rule in CICS 🤕

If not used we can remove, but please double check parser